README.md

```diff
@@ -1,2 +1,182 @@
-# Book-Strore
-This repo is for my bookstore app which I am going to develop using React and Redux 
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h3><b>Bookstore CMS</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Bookstore CMS] <a name="about-project"></a>
+Bookstore is app that I am going to create using react and redux. I will update the descritption respectivily.
+## 🛠 Built With <a name="built-with"></a>
+
+-React
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Version Control</summary>
+  <ul>
+    <li><a href="https://github.com/">Git Hub</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Visual Studio Code</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com">Visual Studio Code</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/html5/">HTML5</a></li>
+    <li><a href="https://www.css3.com/">CSS</a></li>
+  </ul>
+</details>
+ 
+### Key Features <a name="key-features"></a>
+
+- **[Usage of React Libaray]**
+- **[Usge of virtual DOM]**
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- Live demo will be available soon.....
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-Visual Studio Code
+-Github Account
+-Git Bash
+-Node JS
+-React
+### Setup
+
+Clone this repository to your desired folder:
+Use git clone command or downloaded ZIP folder
+Example command:
+
+```sh
+  cd my-folder
+  git clone: (https://github.com/emhamza/book-strore-App)
+```
+
+### Install
+
+Install this project with: npm Example commands:
+
+```sh
+  cd my-project
+  npm init -y
+```
+
+```sh
+  npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x  @babel/plugin-syntax-jsx@7.x  @babel/preset-react@7.x @babel/preset-react@7.x
+```
+
+```sh
+  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
+
+### Usage
+
+To run the project, execute the following command: npm test
+Example command:
+
+```sh
+   GitHub Pages Server
+```
```

Comment:
- [ ] Consider changing this command to `npm run start` as the command you have mentioned is not the correct command to run the server.

Reply:
Certianly, thank you so much for pointing that out!

Reply:
@emhamza Actually I suggested to change line `118`, but you changes line `123`. can you please implement what I just said quickly